packages/flutter/lib/src/material/navigation_bar.dart

```diff
@@ -0,0 +1,1164 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+import 'color_scheme.dart';
+import 'colors.dart';
+import 'elevation_overlay.dart';
+import 'ink_well.dart';
+import 'material.dart';
+import 'material_localizations.dart';
+import 'navigation_bar_theme.dart';
+import 'theme.dart';
+import 'tooltip.dart';
+
+/// Material 3 Navigation Bar component.
```

Comment:
The name we've generally been using is Material 3, rather than v3. The [MDC-Android library](https://github.com/material-components/material-components-android/commit/5c2a5920d511303acc11a74c6d9c863137e5eace) is doing the same so perhaps for consistency we drop the `v`?